main.go

```diff
@@ -200,7 +202,7 @@ func main() {
 		secretsDir:                 *secretsDir,
 		metadataLabels:             nodeMetadata,
 		deploymentInfo:             deploymentInfo,
-		configMesh:                 meshId,
+		configMesh:                 meshID,
 		ipv6Capable:                isIPv6Capable(),
 		gitCommitHash:              gitCommitHash,
 		isTrustedXdsServer:         *isTrustedXdsServer,
```

Comment:
since we are making similar changes, also:
```suggestion
		isTrustedXDSServer:         *isTrustedXDSServer,
```

Reply:
and in the definition please

Reply:
Done. Thanks.